optaplanner-examples/src/main/java/org/optaplanner/examples/vehiclerouting/persistence/VehicleRoutingImporter.java

```diff
@@ -391,6 +391,8 @@ private void createVehicleList() {
                 vehicle.setId(id);
                 id++;
                 vehicle.setCapacity(capacity);
+                //Multiple Depot VRP assign one vehicle for each depot
+                depotList.get(i % depotList.size());
```

Comment:
I fixed it locally, but I am looking into generating a good set multidepot datasets.